packages/odyssey/src/scss/abstracts/_colors.scss

```diff
@@ -21,8 +21,7 @@ $colors: (
     '000': #f5f5f6,
     '100': #ebebed,
     '200': #d7d7dc,
-    '300': #c1c1c8,
-    '400': #aaaab4,
```

Comment:
hmmm, Is this part of our intended public API?

I know that in practice, it is being used as part of the public API in the projects that consume CSS directly from Odyssey because it is published as part of the package.

All that said, this would constitute as a breaking change and require a major version bump. Something to keep in mind as we work toward the more peaceful, just, and verdant world semantic versioning.

Reply:
Yup! We'll be flagging this as a breaking change in our release notes, but only doing a minor version bump (per SemVer, since we're in beta until `1.0`)

Reply:
What is the end goal with colors? Are we going to try to remap `000` to `lightest`, `100` to `light`, etc?

Not having colors for every multiple of 100 seems confusing, as developers who use Odyssey directly need to be aware of what these numbers translate to.

Reply:
Also - this is going to likely mess with some of the re-skin work we've already done with our Courage components, as these will now need to be changed to a new default + retested. 😢 

We can of course hold off on updating Odyssey in Courage, but this kinda forces our hand for major updates. 
FYI ^ @rajaguruduraisamy-okta  and @jeffbelser-okta

Reply:
@jmelberg-okta the goal is to move toward using color maps internally only (e.g. maps for private props, custom properties for public), but we won't have a good/forever solution until our Design Tokens are stronger.

As I ready the release notes, I'll be sure to include upgrade notes + ping @FeliciaWang-okta to ensure Design is prepped as well.